docs/content/guides/author-apps/dapr/how-to-dapr-sidecar/index.md

```diff
@@ -0,0 +1,76 @@
+---
+type: docs
+title: "How-To: Add a Dapr sidecar to Radius"
+linkTitle: "Add a Dapr sidecar to Radius"
+description: "Easily leverage a Dapr sidecar blocks in your application for code and infrastructure portability"
+weight: 100
+categories: "How-To"
+tags: ["Dapr"]
+---
+
+This how-to guide will provide an overview of how to:
+
+- Leverage a [Dapr sidecar](https://docs.dapr.io/concepts/dapr-services/sidecar/) with your Radius Application
+
+## Prerequisites
+
+- [rad CLI]({{< ref getting-started >}})
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Dapr](https://docs.dapr.io/getting-started/install-dapr-cli/)
+
+## Step 1: Define a container
+
+Begin by creating a file named `app.bicep` with a Radius [container]({{< ref "guides/author-apps/containers" >}})
+
+## Step 2: Define the Dapr sidecar extension
+
+Make sure your current Dapr sidecar process is running inside a Kubernetes cluster:
+
+    ```
+    daprd --app-id myapp
+    ```
```

Comment:
I'm a little confused on this. Why is the user interacting with the `daprd` command? And is the app-id already running?

The Dapr extension in Radius should be _creating_ the sidecar and the Dapr app, instead of it being pre-existing.

Could you re-write this section so the user updates the app.bicep file with the Dapr sidecar extension and then they go and see the Dapr sidecar running with either kubectl or the Dapr CLI?